packages/paste-core/components/form-pill-group/__tests__/index.spec.tsx

```diff
@@ -26,6 +26,25 @@ const CustomElementFormPillGroup: React.FC = () => {
   );
 };
 
+const I18nProp: React.FC = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup
+        {...pillState}
+        data-testid="form-pill-group"
+        aria-label="Votre sports favoris:"
+        i18nKeyboardControls="Appuyez sur Supprimer ou Retour arrière pour supprimer. Appuyez sur Entrée pour basculer la sélection."
```

Comment:
thought (non-blocking): reading this making me realize just how rusty my french is. :P